### ISSUE TITLE
Essentia Provider will not provide Essentia to Thaumatorium

### ISSUE THREAD

**Issue by Xevion** [Nividica/ThaumicEnergistics#346]
Modpack: (slightly modified) FTB Infinity Evolved: Expert Mode
Version: MC 1.7.10
Mod Version: Thaumic Energistics 1.1.3.0

I've been trying to fix this for a couple hours now
I've gone through all the relevant issues, couldn't find anything.
The essentia provider will NOT expend relevant essentia from my ME setup (in both my real and test world) to a Thaumatorium (Alchemical Construct + Crucible)

I have tried
- Updating to the current release version (rv3 stable release)
- Using pipes on every side
- Using the Provider on every side directly
- Rebuilding the Provider and Thaumatorium in differnet orders, times etc.
- Searching for answers on Google about this
- Exporting Essentia won't work either it seems

GUI
![image](https://user-images.githubusercontent.com/22359810/41392826-9084e886-6f68-11e8-87d9-41ba88eb08f1.png)

Setup
![image](https://user-images.githubusercontent.com/22359810/41392818-8a686d38-6f68-11e8-86ff-0fee4a44d234.png)

Essentia Terminal 
![image](https://user-images.githubusercontent.com/22359810/41392801-7adf73d4-6f68-11e8-9afa-46d7d9b0d33d.png)

Similar setup in my Real world, I'm really unsure of what to do.
I hope you're still active in answer/helping with problems like this. :C

**Comment by Nividica** [comment]
Since you are working with a Thaumatorium, my first question is always, did you remember to put a heat source under it? It wont bother requesting essentia until it has heat.

**Comment by Xevion** [comment]
After I got off and posted this, I was up at night, couldn't go to sleep because I realized since it had a crucible, it must need a heat source. I'm serious, I just sat there thinking to myself "my god am I stupid".

So, yes, the problem was, simply put, me not putting a heat source (netherrack, nitor) under the Crucible/Thaumatorium.

But thanks for answering, no one I asked could get me an answer no matter what I asked.
Great mod btw.
